src/node.cc

```diff
@@ -235,6 +237,7 @@ static struct {
     // Destroy tracing after the platform (and platform threads) have been
     // stopped.
     tracing_agent_.reset(nullptr);
+    trace_state_observer_.reset(nullptr);
```

Comment:
Should we call `controller->RemoveTraceStateObserver(trace_state_observer_.get())` here?

Reply:
I’m not sure, but does the controller still exist at this point?